bats/foreign-keys.bats

```diff
@@ -47,6 +47,16 @@ SQL
     [[ `echo "$output" | tr -d "\n" | tr -s " "` =~ 'CONSTRAINT `fk_named` FOREIGN KEY (`v1`) REFERENCES `parent` (`v1`)' ]] || false
 }
 
+@test "foreign-keys: parent table index recquired" {

```

Comment:
_recquired_

Reply:
rekwired

Reply:
I would like the record to show that neither of my coworkers were helpful in fixing my spelling error